terraform/cluster/aws/main.tf

```diff
@@ -57,6 +57,7 @@ resource "aws_eks_cluster" "cluster" {
 
   name     = var.name
   role_arn = aws_iam_role.cluster.arn
+  tags     = local.tags
```

Comment:
I'm not sure if you already tried that, but seems like applying the tags to the AWS provider is cleaner(it will propagate to all created resources)
https://registry.terraform.io/providers/hashicorp/aws/latest/docs#default_tags-configuration-block

Reply:
@heronrs I checked it misses the ec2 tagging

Reply:
Should we explicitly declare `tags` in the `locals` block or how is it getting the values? Or should it be `var.tags`?

Reply:
It's in the another file https://github.com/convox/convox/blob/d2b22a9cceeceb80fa0a552fcb61ab075cdd6657/terraform/cluster/aws/network.tf#L3

since it were there before, so i didn't change it